run-trino-env.sh

```diff
@@ -1,17 +1,17 @@
-#!/bin/sh
+#!/bin/bash
 
-trinoCloneDir=${1:-~/repositories/trino}
+trinoCloneDir=${1:-~/repositories/cccs-work/trino}
```

Comment:
Is this cccs-work folder structure widely-spread enough that we want to make it the default for everyone? I certainly don't use it...

Reply:
Well that is the structure recommended in the "So You're new to Dasi2C"  document!

Reply:
Ah! I did not realize that. I'm also not sure why... I don't do anything on my laptop that *wouldn't* qualify as "cccs-work". But ok! 🤷